charts/argo-cd/templates/_helpers.tpl

```diff
@@ -137,6 +137,10 @@ helm.sh/chart: {{ include "argo-cd.chart" .context }}
 {{ include "argo-cd.selectorLabels" (dict "context" .context "component" .component "name" .name) }}
 app.kubernetes.io/managed-by: {{ .context.Release.Service }}
 app.kubernetes.io/part-of: argocd
+app.kubernetes.io/version: {{ .context.Chart.AppVersion | quote }}
```

Comment:
We cannot use the version in the template function. The version label is handled separately in every deployment.yaml